uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/JolokiaJMXNavigatorTest.java

```diff
@@ -0,0 +1,248 @@
+import org.eclipse.reddeer.eclipse.ui.console.ConsoleView;
+import org.eclipse.reddeer.eclipse.ui.views.log.LogView;
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.requirements.cleanworkspace.CleanWorkspaceRequirement.CleanWorkspace;
+import org.eclipse.reddeer.requirements.openperspective.OpenPerspectiveRequirement.OpenPerspective;
+import org.eclipse.reddeer.swt.condition.ShellIsAvailable;
+import org.eclipse.reddeer.swt.impl.menu.ContextMenuItem;
+import org.eclipse.reddeer.swt.impl.toolbar.DefaultToolItem;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.eclipse.reddeer.workbench.handler.WorkbenchShellHandler;
+import org.eclipse.reddeer.workbench.impl.shell.WorkbenchShell;
+import org.eclipse.reddeer.workbench.ui.dialogs.WorkbenchPreferenceDialog;
+import org.jboss.tools.fuse.reddeer.LogGrapper;
+import org.jboss.tools.fuse.reddeer.ProjectTemplate;
+import org.jboss.tools.fuse.reddeer.ProjectType;
+import org.jboss.tools.fuse.reddeer.SupportedCamelVersions;
+import org.jboss.tools.fuse.reddeer.condition.JMXConnectionIsAvailable;
+import org.jboss.tools.fuse.reddeer.launchconfigurations.LocalCamelContextLaunchConfiguration;
+import org.jboss.tools.fuse.reddeer.launchconfigurations.LocalCamelContextLaunchConfigurationJRETab;
+import org.jboss.tools.fuse.reddeer.perspectives.FuseIntegrationPerspective;
+import org.jboss.tools.fuse.reddeer.preference.ConsolePreferencePage;
+import org.jboss.tools.fuse.reddeer.projectexplorer.CamelProject;
+import org.jboss.tools.fuse.reddeer.view.FuseJMXNavigator;
+import org.jboss.tools.fuse.reddeer.wizard.JMXNewConnectionWizard;
+import org.jboss.tools.fuse.reddeer.wizard.JMXNewConnectionWizard.ConnectionType;
+import org.jboss.tools.fuse.ui.bot.tests.utils.ProjectFactory;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@CleanWorkspace
+@OpenPerspective(FuseIntegrationPerspective.class)
+@RunWith(RedDeerSuite.class)
+public class JolokiaJMXNavigatorTest {
+
+	private static final String PROJECT_NAME = "camel-blueprint";
+	private static final String PROJECT_CAMEL_CONTEXT = "src/main/resources/OSGI-INF/blueprint/blueprint.xml";
+	private static final String JOLOKIA_CONNECTION_NAME = "test-blueprint-connection";
+
+	private static Logger log = Logger.getLogger(JMXNavigatorTest.class);
+
+	/**
+	 * Prepares test environment
+	 */
+	@BeforeClass
+	public static void setupCreateProject() {
+
+		log.info("Maximizing workbench shell.");
+		new WorkbenchShell().maximize();
+
+		log.info("Disable showing Console view after standard output changes");
+		WorkbenchPreferenceDialog dialog = new WorkbenchPreferenceDialog();
+		ConsolePreferencePage consolePref = new ConsolePreferencePage(dialog);
+		dialog.open();
+		dialog.select(consolePref);
+		consolePref.toggleShowConsoleErrorWrite(false);
```

Comment:
Use direct API for setting all preferences (except the cases where we test the preference dialog/page)